libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -126,8 +127,8 @@ export default function TeamCharacterSelector({
           ) : undefined
         }
       >
-        <CardContent sx={{ display: 'flex', justifyContent: 'center', pb: 0 }}>
-          <CardActionArea>
+        <CardContent sx={{ display: 'flex', justifyContent: 'center', p: 0}}>
+          <CardActionArea sx={{ px: 2, pt: 2}}>
```

Comment:
On second thought, just change this to
```suggestion
          <CardActionArea sx={{ p: 2 }}>
```